202203/sc 4node setup (#403)
klaytn/klaytn-docs: docs/node/service-chain/getting-started/4nodes-setup-guide.md

$ export PATH=$PATH:~/path/to/kscn-XXXXX-amd64/bin
```

> SCN also provides RPM Distribution(RHEL/EntOS/Fedora). For more information, please refer [here](../references/scn/installation.md).

Reviewer: ```suggestion
SCN also provides RPM Distribution(RHEL/EntOS/Fedora). For more information, please refer to [Installation](../references/scn/installation.md).
```

> SCN also provides RPM Distribution(RHEL/EntOS/Fedora). For more information, please refer to [Installation](../references/scn/installation.md).

Reviewer: ```suggestion
SCN also provides various RPM distributions like RHEL, CentOS, and Fedora. For more information, please refer to [Installation](../references/scn/installation.md).
```

```console

> # curl -o /etc/yum.repos.d/klaytn.repo https://packages.klaytn.net/config/rhel/7/prod.repo

Reviewer: ```suggestion
$ curl -o /etc/yum.repos.d/klaytn.repo https://packages.klaytn.net/config/rhel/7/prod.repo
```

I think we've used '$' instead of '#'?

Author: Can I commit this change?

  % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed
     100 118 100 118 0 0 1113 0 --:--:-- --:--:-- --:--:-- 1102 

> # yum list | grep klaytn 

Reviewer: ```suggestion
$ yum list | grep klaytn 
```

Author: I will commit this through git.

packages-klaytn-prod 31 kB/s | 2.9 kB 00:00 
homi.x86_64           v1.8.0-0.el7      packages-klaytn-prod 
kbnd.x86_64           v1.8.0-0.el7      packages-klaytn-prod 
kcnd.x86_64           v1.8.0-0.el7      packages-klaytn-prod 
kcnd-baobab.x86_64    v1.8.0-0.el7      packages-klaytn-prod 
kend.x86_64           v1.8.0-0.el7      packages-klaytn-prod 
kend-baobab.x86_64    v1.8.0-0.el7      packages-klaytn-prod 
kgen.x86_64           v1.8.0-0.el7      packages-klaytn-prod 
kpnd.x86_64           v1.8.0-0.el7      packages-klaytn-prod 
kpnd-baobab.x86_64    v1.8.0-0.el7      packages-klaytn-prod 
kscnd.x86_64          v1.8.0-0.el7      packages-klaytn-prod 
ksend.x86_64          v1.8.0-0.el7      packages-klaytn-prod 
kspnd.x86_64          v1.8.0-0.el7      packages-klaytn-prod 

> # yum install kscnd

Reviewer: ```suggestion
$ yum install kscnd
```

Author: I will commit this through git.

```

## Step 1: Create genesis.json and nodekeys <a id="step-1-create-genesis-json-and-a-key"></a>

We will use homi utility to generate the needful files.

> Homi is a utility that automatically generates scripts, configuration files, and private keys necessary to configure the Klaytn blockchain.

Reviewer 2: ```suggestion
`homi` is a utility that automatically generates scripts, configuration files, and private keys necessary to configure the Klaytn blockchain.
```
